odoo_module_migrate/migration.py

```diff
@@ -89,9 +89,23 @@ def __init__(
         for module_name in module_names:
             self._module_migrations.append(ModuleMigration(self, module_name))
 
+        if os.path.exists(".prettierrc.yml"):
+            self._run_prettier(module_names)
+
         # get migration scripts, depending to the migration list
         self._get_migration_scripts()
 
+    def _run_prettier(self, module_names):
+        logger.info("Run prettier")
+        _execute_shell(
+            "pre-commit run -a", path=self._directory_path, raise_error=False)
+        logger.info("Add and commit change done by prettier")
+        _execute_shell("git add -A", path=self._directory_path)
+        _execute_shell(
+            "git commit -m '[IMP] %s: black, isort, prettier'  --no-verify"
+            % ", ".join(module_names),
+            path=self._directory_path)
```

Comment:
This doesn't respect  ``--no-commit`` option

Reply:
Done, I test if commit is enable